content/about.md

```diff
@@ -11,9 +11,9 @@ date: 2020-04-06T03:38:58+05:00
 ---
 ## Work
 - [Advocate](https://docs.fedoraproject.org/en-US/mindshare-committee/advocate/) @ [Fedora Project](https://getfedora.org/)
-- Engineering Intern @ [Sastaticket.pk](https://www.sastaticket.pk/)
+- DevOps Intern @ [Nisum.pk](https://www.Nisum.pk/)
```

Comment:
@bilalraiss , Please Fix Nisum HTTP Address, It's: `http://nisum.com/`

And Update `Nisum.pk` to `Nisum` in text.

Reply:
sure